apps/condo/domains/common/components/trackers/AmplitudeInstance.ts

```diff
@@ -37,7 +37,14 @@ class AmplitudeInstance extends TrackerInstance {
     }
 
     logInstanceEvent (trackerLogEventProps: ITrackerLogEventType): void {
-        this.instance.setUserProperties(trackerLogEventProps.userProperties)
+        const Identify = this.instance.Identify
```

Comment:
It is unnecessary. Because of this method is not callable from class instance.
Logic of the `logInstanceEvent` call restrictions implemented inside base class `TrackerInstance`

![image](https://github.com/open-condo-software/condo/assets/25844927/d5014950-c5b9-420f-8a73-ef771c648d6d)

For more detailed information you can check out `apps/condo/domains/common/components/trackers/TrackerInstance.ts`